crates/chain/src/chain_data.rs

```diff
@@ -99,44 +47,31 @@ impl TxHeight {
 )]
 pub enum ConfirmationTime {
     Confirmed { height: u32, time: u64 },
-    Unconfirmed,
+    Unconfirmed { last_seen: u64 },
```

Comment:
Can we document each variant. Specify the units of last_seen and time.

It's time to add `#[warn(missing_docs)]` on this project and make sure CI is failing on it. Fine to do it in a followup PR but i think it should be done before making the next alpha release.

Reply:
Documentation and `#[warn(missing_docs)]` added in 250458ae01e7157c97f3c88f572e4d0ff21c019f